src/main/java/space/obminyashka/items_exchange/security/jwt/refresh/RefreshTokenResponse.java

```diff
@@ -0,0 +1,17 @@
+package space.obminyashka.items_exchange.security.jwt.refresh;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class RefreshTokenResponse {
+
+    private static final String TOKEN_TYPE = "Bearer";
```

Comment:
You can use BEARER_PREFIX from JwtTokenProvider for that purpose instead.

Reply:
Ok